css/freelancer.min.css

```diff
@@ -2,4 +2,8 @@
+
+
+/* I think there's an issue with the width of the footer vs the width of the wrapper div (ball pit) that is above it. If I resize my screen, the footer doesn't extend to the edge of screen whereas the wrapper div does. */
+/* The counter that tracks how many balls aren't moving doens't seem to be incrementing even though I can see that there are balls that aren't moving. */
```

Comment:
Yeah i had a lot of trouble getting the canvas to match the container it is in since it constantly is redrawing itself. Even tagging its size and doing it more dynamically based on auto or relative width to parent wasnt working for me. I've read a bit more about it and was gonna give it another go!
The counter only runs when you press it! I wanted to keep it a game more then anything else